source/ui/BioXAS/BioXASXASScanConfigurationView.cpp

```diff
@@ -20,8 +20,8 @@ const AMScanConfiguration* BioXASXASScanConfigurationView::configuration() const
 
 void BioXASXASScanConfigurationView::setConfigurationName(BioXASXASScanConfiguration *configuration, const QString &newName)
 {
-	if (configuration && configuration->userScanName() != newName)
-		configuration->setUserScanName(newName);
+	if (configuration)
```

Comment:
No major reason. The same check is performed in `AMDbObject::setName()` already, don't need to have a duplicate check here.